notebooks/en/enterprise_dedicated_endpoints.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<p>This can probably be a second level section <code>## Additional information</code> since it is pretty general and not specific to creating endpoints for a variety of models. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>